### PR TITLE
Change user name formatting

### PR DESCRIPTION
**Pull request by mhinz**
Hopefully for the last time..

- For normal messages our Gitter users chose to use `code blocks` instead of **bold** for user name formatting.
- Make all `/me` messages *italic*.

### CODE REVIEW

File: lib/BridgedRoom.js [hunk 1]
```diff
@@ -111,11 +111,11 @@ BridgedRoom.prototype.onMatrixMessage = function(message) {
   // gitter supports Markdown. We'll use that to apply a little formatting
   // to make understanding the text a little easier
   if (message.content.msgtype == 'm.emote') {
-    var text = '**' + from + '** ' + message.content.body;
+    var text = '*' + (from + ' ' + message.content.body).replace(/\*/g, '\\*') + '*';
     this._gitterRoom.sendStatus(text);
   }
   else {
-    var text = '**<' + from + '>**: ' + message.content.body;
+    var text = '`" + from + '` ' + message.content.body;
```
Comment (leonerd) [inline lib/BridgedRoom.js]: Not quite sure what this lone unmatched `"` symbol is doing here...

Comment (mhinz) [inline lib/BridgedRoom.js]: One of my typical late-night-commits..

**Comment by leonerd** [comment]
At some point I might make message formatting configurable per room. But currently since neovim is basically the only room around that's using this I don't yet see a point. So I'll just accept this for now.

**Comment by mhinz** [comment]
That makes sense. Thanks!

**Pull request merged**